### DISCUSSION TITLE
Why are mainstream Spanish TV channels not included?

### DISCUSSION THREAD

**Discussion by pablohoffman** [iptv-org/community#464] [Q&A] [answered]
I noticed there are many local/niche channels in the Spanish playlist, but the mainstream country-wide channels (eg. La Sexta, Antena 3...) are not included. Is there a reason (legal or otherwise) for this?

**Comment by PopeyeTheSai10r** [comment, marked as answer]
Any channel that is **not** on the [blocklist](https://github.com/iptv-org/database/blob/master/data/blocklist.csv) can be added to the iptv repository as long as someone has the stream link for it.

You can either create a pull request or create an issue if you [issue](https://github.com/iptv-org/iptv/issues/new/choose) if you have a link to the stream or would like to request a channel.
